examples/k8s/otel-config.yaml

```
@@ -12,7 +12,9 @@ data:
       otlp:
         protocols:
           grpc:
+            endpoint: ${MY_POD_IP}:4317
```

Can you use the new `env` syntax instead? So e.g. here it would be

```suggestion
            endpoint: ${env:MY_POD_IP}:4317
```

---

(Helm chart will eventually do the same, see open-telemetry/opentelemetry-helm-charts/issues/761 )